Podcast/EpisodeSubjectView.swift

```diff
@@ -121,7 +121,8 @@ class EpisodeSubjectView: UIView {
         }
 
         descriptionLabel.snp.makeConstraints { make in
-            make.top.equalTo(podcastImage.snp.bottom).offset(descriptionLabelOffset)
+            make.top.greaterThanOrEqualTo(podcastImage.snp.bottom).offset(descriptionLabelOffset)
+            make.top.greaterThanOrEqualTo(dateTimeLabel.snp.bottom).offset(descriptionLabelOffset)
```

Comment:
nice!!

Reply:
i'm still getting constraint errors on this line though with the `EpisodeUtilityBarButton` height